### ISSUE TITLE
Terminal input not working with ConPTY disabled

### ISSUE THREAD

**Issue by elkondzik** [Eugeny/terminus#3397]
Version: 1.0.130
Platform: win32 6.1.7601
Plugins: none

### Whenever key is pressed in the terminal, app is throwing following exception:

`
events.js:292 Uncaught Error: This socket has been ended by the other party at Socket.writeAfterFIN [as write] (net.js:456) at WindowsTerminal._doWrite (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\@terminus-term\node-pty\lib\windowsTerminal.js:114) at WindowsTerminal._defer (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\@terminus-term\node-pty\lib\windowsTerminal.js:156) at WindowsTerminal._write (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\@terminus-term\node-pty\lib\windowsTerminal.js:111) at WindowsTerminal.Terminal.write (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\@terminus-term\node-pty\lib\terminal.js:81) at Session.write (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\builtin-plugins\terminus-terminal\dist\index.js:12988) at TerminalTabComponent.sendInput (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\builtin-plugins\terminus-terminal\dist\index.js:8736) at SafeSubscriber._next (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\builtin-plugins\terminus-terminal\dist\index.js:8964) at SafeSubscriber.__tryOrUnsub (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\rxjs\internal\Subscriber.js:205) at SafeSubscriber.next (C:\Users\xxx\AppData\Local\Programs\Terminus\resources\app.asar\node_modules\rxjs\internal\Subscriber.js:143)
`
[Full_Debug.log](https://github.com/Eugeny/terminus/files/5903633/Full_Debug.log)

**Comment by Eugeny** [comment]
See if enabling ConPTY under Settings -> Shell helps

**Comment by Kri-Ol** [comment]
Same here. Windows 2016 Server (roughly the same as Windows 10 1607), no ConPTY support

**Comment by radiaku** [comment]
Bump up, I got same error. and no conpty support

**Comment by elkondzik** [comment]
Yes, exactly. No ConPTY property under settings->shell.
Any other suggestions? Or is this bug to be fixed?
Oh, didn't mentioned OS: Windows 7

**Comment by GanapathyRaman** [comment]
> See if enabling ConPTY under Settings -> Shell helps

Thanks. Enabling ConPTY works.

**Comment by Eugeny** [comment]
Upstream issue: https://github.com/microsoft/node-pty/issues/457

**Comment by elkondzik** [comment]
# Changed release from 1.0.130 --> 1.0.132
## Nothing has changed regarding keyboard input on Windows 7:

1. **SYSTEM**
   *  All releases are OK on Windows 10 (i.e. keyboard works in terminal)
   * None of the releases are OK on Windows 7
2. **ConPTY**
   * ConPTY option is still not available in 1.0.132 in Settings --> Shell
   * Setting ConPTY as variable to: true/false/enabled/disabled for any release doesn't help
   
### Any plans fixing it in the nearest future? :)

**Comment by uluhonolulu** [comment]
Windows Server 2016, no ConPTY version.

In addition to the socket error, I'm seeing this:
```
C:\Users\asmirnov\AppData\Local\Programs\Terminus\resources\builtin-plugins\terminus-core\dist\index.js:5724 [updater] Error: Can not find Squirrel
    at AutoUpdater.checkForUpdates (electron/js2c/browser_init.js:17:1405)
    at electron/js2c/browser_init.js:225:8560
    at IpcMainImpl.<anonymous> (electron/js2c/browser_init.js:225:4947)
    at IpcMainImpl.emit (events.js:315:20)
    at IpcMainImpl.EventEmitter.emit (domain.js:486:12)
    at Object.<anonymous> (electron/js2c/browser_init.js:161:10389)
    at Object.emit (events.js:315:20)
    at Object.apply [as emit] (domain.js:486:12)
    at Object.n.emit (C:\Users\asmirnov\AppData\Local\Programs\Terminus\resources\app.asar\dist\webpack:\terminus\node_modules\@sentry\electron\dist\main\integrations\electron.js:66:25)
```

Hope that helps!

P.S. Thank you for this wonderful program! I started to love command line after I started using it!
